frontend/src/plaquettes/Template.js

```diff
@@ -1,230 +1,87 @@
-import { Container, Graphics, Color } from 'pixi.js';
+import { Container, Graphics } from 'pixi.js';
 import Plaquette from './Plaquette';
 import notification from '../components/notification';
 import Button from '../components/Button';
 
+/**
+ * Template class to create the template for the plaquettes
+ * @class Template
+ * @param {Container} workspace - The workspace container
+ * @param {PIXI.Application} app - The PIXI application
+ * @param {number} x - The x coordinate
+ * @param {number} y - The y coordinate
+ * @returns {void}
+ */
 export default class Template {
-  constructor(selectedQubits, workspace, plaquetteButton, app) {
+  constructor(workspace, app, x, y) {
     // UI Properties
     this.app = app;
     this.container = new Container();
     this.buttonMode = true;
     this.cursor = 'pointer';
     this.mode = 'static';
     this.isDragging = false;
+    this.x = x;
+    this.y = y;
     this.startX = 0;
     this.startY = 0;
     this.plaquette = null;
-    this.plaquetteButton = plaquetteButton;
-    this.templateQubits = selectedQubits || [];
+    this.selectQubitsButton = new Button('Select 3 qubits to make a Plaquette', x, y + 50);
```

Comment:
Before this point, qubits should not be clickable (when my mouse hovers over them, the Mickey Mouse hand should not appear). Can we change that?

Reply:
Also, it should say "Select 3+ qubits to make a Plaquette Footprint"

Reply:
Done

Reply:
the clicking functionality resides in the qubit class and we can do it in a separate issue.